frontend/src/middleware.ts

```diff
@@ -0,0 +1,13 @@
+import { NextResponse } from 'next/server';
+import type { NextRequest } from 'next/server';
+
+export function middleware(request: NextRequest) {
+	let cookie = request.cookies.get('ods_login_cookie_nomnom');
+	if (!cookie) {
+		return NextResponse.redirect(new URL('/login', request.url));
+	}
+}
+
+export const config = {
```

Comment:
could we name this better?

Reply:
<a href="https://nextjs.org/docs/app/building-your-application/routing/middleware#matcher">No</a>